tutorials/rnn/ptb/ptb_word_lm.py

```diff
@@ -225,15 +225,15 @@ def make_cell():
 
     self._initial_state = cell.zero_state(config.batch_size, data_type())
     state = self._initial_state
-    # Simplified version of tensorflow_models/tutorials/rnn/rnn.py's rnn().
+    # Simplified version of tf.nn.static_rnn().
     # This builds an unrolled LSTM for tutorial purposes only.
-    # In general, use the rnn() or state_saving_rnn() from rnn.py.
+    # In general, use tf.nn.static_rnn() or tf.nn.static_state_saving_rnn().
```

Comment:
I think it's in contrib, not in nn, no?

Reply:
Either can work well, and ```contrib``` only import the API and implementation from ```nn```. So I think ```nn``` should be preferred. Does it reasonable?